bundle.yaml

```diff
@@ -26,8 +26,17 @@ applications:
     num_units: 1
     constraints: mem=2048
     options:
-      plugin_plpython3u_enable: True
-      experimental_max_connections: 500
+      plugin_plpython3u_enable: true
+      plugin_ltree_enable: true
+      plugin_intarray_enable: true
+      plugin_debversion_enable: true
+      plugin_pg_trgm_enable: true
+  pgbouncer-lds:
+    charm: pgbouncer
+    channel: 1/edge
+    options:
+      pool_mode: transaction
+      max_db_connections: 40
```

Comment:
Could you comment the reasoning for these settings please?

Reply:
That number is how many connections are always open to be used pgbouncer for it's pool. It cannot be greater than the max connections of postgres which is 100 in this case.

Reply:
"You don't want default_pool_size to be equal to max_connections. That would leave no connections left over for things like monitoring, reporting, or debugging, or anything else. So back off a few. "

Reply:
Thanks!